documentation/shared/attributes.adoc

```diff
@@ -13,8 +13,8 @@
 :toclevels: 3
 
 //Latest Strimzi version
-:ProductVersion: 0.47.0
-:ProductVersionPrevious: 0.46.0
+:ProductVersion: 0.46.0
+:ProductVersionPrevious: 0.45.0
```

Comment:
This is wrong -> this is for the next release so it should be 0.47.0 and 0.46.0.

Reply:
It was that way but I thought it was a mistake I made before, before ProductVersion was, for me, the current already out version and 0.47.0 is not out (of course). I will revert the change.

Reply:
Yeah, it can be a bit confusing I guess. But unlike the examples, install files, and helm charts, the docs are there only once. So that is why they are already for 0.47 in the main branch while you are only adding the 0.46 release files.